packages/manager/apps/dedicated/client/app/dedicated/server/server.service.js

```diff
@@ -2139,9 +2140,10 @@ export default class ServerF {
   getSms(productId) {
     let promises = [];
 
-    if (this.coreConfig.getRegion() === 'CA') {
+    if (includes(['CA', 'US'], this.coreConfig.getRegion())) {
```

Comment:
Looks good to me as an hotfix.

Could be turned into `Array.prototype.includes()` I think to ease the drop support of Internet Explorer at some point.

Thank you!